### PR TITLE
Add documentation about alternative ways to delete clusters

### PR DESCRIPTION
**Pull request by sean-abbott**
### Description

I added a couple of lines to help people delete the EKS cluster in the event of failures.

### Checklist

- [x] Added/modified documentation as required (such as the `README.md`, and `examples` directory)

### CODE REVIEW

File: site/content/usage/21-troubleshooting.md [hunk 1]
```diff
@@ -45,3 +45,6 @@ vpc:
       us-east-1a: {id: subnet-33333333}
       us-east-1b: {id: subnet-44444444}
 ```
+
+### Deletion issues
+If your delete does not work, or you forget to add -wait on the delete, you may need to go to use amazon's other tools to delete the cloudformation stacks. This can be accomplished via the gui or with the aws cli.
```
Comment (martina-if) [inline site/content/usage/21-troubleshooting.md]: nit: surround `-wait` with  ` ` `

Comment (martina-if) [reply site/content/usage/21-troubleshooting.md]: "you may need to go to use amazon's other tools" -> "you may need to use amazon's other tools"

**Comment by martina-if** [comment]
Hi @sean-abbott , thank you for contributing to eksctl!

File: site/content/usage/21-troubleshooting.md (hunk 1 above)
Comment (sayboras) [reply site/content/usage/21-troubleshooting.md]: should it be `--wait` ? :)

**Comment by martina-if** [comment]
Hi @sean-abbott , gentle reminder :) Can you apply the small fix and rebase so we can merge this? :) This will be useful for other users!

**Comment by martina-if** [comment]
Hi @sean-abbott Do you have time to make the small fix and the rebase of this PR?

**Comment by sean-abbott** [comment]
Yup, sorry. Done.

**Pull request merged**